lib/exit.js

```diff
@@ -115,7 +115,7 @@ export default class Exit {
         plasmaChain.eth.getTransaction(
           util.bufferToHex(tx.inputs[0].prevout.hash),
           signer
-          ? Promise.resolve(signer.signMessage(sigHash))
+          ? signer.signMessage(sigHash)
```

Comment:
change Signer interface to return Promises

https://github.com/leapdao/burner-wallet/pull/24/commits/4c84931b73baf967797aae1016cc544279569f23